src/plugindata.h

```diff
@@ -379,7 +392,7 @@ GeanyProxyProbeResults;
 struct GeanyProxyFuncs
 {
 	/** Called to determine whether the proxy is truly responsible for the requested plugin.
-	 * A NULL pointer assumes the probe() function would always return @ref PROXY_MATCHED */
+	 * A NULL pointer assumes the probe() function would always return @ref GEANY_PROXY_MATCH */
 	gint		(*probe)     (GeanyPlugin *proxy, const gchar *filename, gpointer pdata);
```

Comment:
ideally the function would return `GeanyProxyProbeResults`, but well, it'd technically be an ABI break -- even though we know everyone use 2's complement so it wouldn't change nothing, but meh.